server/pbench/bin/pbench-report-status

```diff
@@ -93,8 +93,8 @@ fi
 
 # Snip off the trailing "-<TZ>" - assumes that <TZ> does not contain a "-"
 datestamp="${timestamp%-*}"
-# Snip off the time part of the timestamp
-datestamp=${datestamp%T*}
+# Snip off the time part of the timestamp to get YYYY-MM
+datestamp=${datestamp%-*}
```

Comment:
OK - I see. I went back far enough into the history and saw what you were talking about. So who is producing illegal index names?